docs/docs/configuration/tooltip.md

```diff
@@ -197,7 +197,13 @@ The tooltip items passed to the tooltip callbacks implement the following interf
     datasetIndex: number,
 
     // Index of this data item in the dataset
-    dataIndex: number
+    dataIndex: number,
+
+    // X position of matching point
+    x: number,
```

Comment:
this will need to be updated

Reply:
done (also fixed a `this._chart` I noticed there)